index.html

```diff
@@ -5,12 +5,95 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="../src/styles.css" />
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
-    <!-- Add your markup here -->
-</body>
+    <header>
+        <nav class="container container-flex header">
+          <img src="../images/sumo1.jpg" alt="sumo" class="logo" />
+          <label for="check" class="check-btn">
+            <i class="fa fa-bars"><p class="checkbtn-text">menu</p></i>
+          </label>
+  
+          <h2 class="shop-title">sumo bakery</h2>
+          <input type="checkbox" id="check" />
+          
+          <ul class="nav-bar nav-list">
+            <li class="nav-list-item"><a href="" class="nav-link">home</a></li>
+            <li class="nav-list-item"><a href="" class="nav-link">order</a></li>
+            <li class="nav-list-item"><a href="#service" class="nav-link">service</a></li>
+            <li class="nav-list-item"><a href="" class="nav-link">cake</a></li>
+            <li class="nav-list-item"><a href="" class="nav-link">feedback</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      
+          
+          <section id="service"  class="container container-flex hero-area"> 
+            
+              <aside >
+                  <h1>HELLO</h1>
+                  <p> Lorem ipsum, <strong>SUMO like cake a lot </strong>  dolor sit amet
+                     consectetur adipisicing elit. Eveniet, provident excepturi? Illum veritatis, 
+                     ullam, amet vero debitis maiores velit ipsa sit non, neque itaque eius quibusdam 
+                     consequatur in incidunt tenetur quia. Id et deleniti doloribus modi quas ipsum esse nam. 
+                    Lorem ipsum do <strong>He make Cake with ......</strong>, amet consectetur adipisicing elit. Reprehenderit
+                     molestiae    praesentium tempore saepe eos, quo eligendi nobis. Nemo, magnam ut. <a  class="article-info"  href=""> Read More </a></p>
+                  
```

Comment:
As a rule, if clicking something _doesn't_ take you to another page, then use the `<button>` element instead.

Reply:
thank you , i'll change to button .